<launch>

<!-- ===== Compute traversability from pointCloud ==================== -->
<node name="mesh_to_traversability_pcd_node" pkg="mesh_to_traversability_pcd" type="mesh_to_traversability_pcd_node" output="screen" >

<param name="load_from_file" value="true" />
<param name="file" value="/home/eth/Partage_VM/Tradr-Montelebretti-Review-share/PLY-UAV/surface-L2-clean-hand-aligned.ply"/>
<!-- <param name="file" value="$(find mesh_to_traversability_pcd)/pcd/mesh_grass_zurich.pcd"/> -->

<param name="verbose" value="true"/>
<param name="z_threshold" value="0.5"/>
<param name="image_scale" value="20"/>
<param name="automatic_pub" value="true"/>
</node>


<!-- ===== RVIZ =================================== -->
<node name="rviz" pkg="rviz" type="rviz" args="-d $(find mesh_to_traversability_pcd)/rviz/load_pcd.rviz"/>

</launch>
